code/script.js

```diff
@@ -1,9 +1,10 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
 function toggle() {
-  this.classList.toggle("active")
```

Comment:
You can use this one if you want to use the  .active + .answer selector to toggle the accordions visibility to hidden in the start se comment on lie 74